packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -584,6 +584,42 @@ const ruleset: any = {
       },
     },
 
+    ///
+    /// ARM RPC rules for SystemData
+    ///
+    // RPC Code: RPC-SystemData-V1-01
+    // Note that this rule is normally already covered when the common types Resource is used.
+    // Here we are still checking for usage of common types system data for the scenarios where
+    // RP teams have already defined Resource and are unable to switch to the common types resource
```

Comment:
> defined Resource and are unable to switch to the common types resource [](http://example.com/codeflow?start=28&length=71)

Shouldnt we be checking the actual definition in that case? I see the main thing being checked if the common-type is being used here. Is my understanding correct?

Reply:
We previously discussed that I should also create common types for tracked and proxy resources without the system data so that teams without system data can still use common types. I started that [here](https://github.com/Azure/azure-rest-api-specs/pull/23341/files) but still need to finish that PR. After that, we decided that we can just check that common types are used here.

Reply:
Yes, that is correct. The idea here is that by ensuring the common-types definition for systemData is used, you are ensuring that the body is correct.